docs/basics/complete_example.md

```diff
@@ -20,11 +20,15 @@ We're going to take in a dataframe, and return a dataframe of the same type.
 Therefore, we use `@nw.narwhalify_method` (the counterpart to `@nw.narwhalify` which is
 meant to be used for methods):
 
+!!! info
+    As of version 0.y.z, `@nw.narwhalify_method` has been deprecated, and `nw.narwhalify`
```

Comment:
Reminder for future self: Change version

I didn't want to pin it as we ship very fast, and this PR could stay here for longer than expected

Reply:
from https://github.com/search?q=narwhalify_method&type=code, it looks like nobody's using it

maybe we can just get rid of it right away

Reply:
It looks as if it never existed now 😂